app/Domain/Tickets/Js/ticketsController.js

```diff
@@ -1222,7 +1222,54 @@ leantime.ticketsController = (function () {
                         { "visible": false, "targets": 10 },
                         { "visible": false, "targets": 11 },
                         { "target": "no-sort", "orderable": false},
-                    ]
+                    ],
+                "footerCallback": function ( row, data, start, end, display ) {
+                    var api = this.api(), data;
+
+                    // converting to interger to find total
+                    var intVal = function ( i ) {
+                        return typeof i === 'string' ?
+                            i.replace(/[\$,]/g, '')*1 :
+                            typeof i === 'number' ?
+                                i : 0;
+                    };
+
+                    // computing column Total of the complete result
+
+
+                    var plannedHours = api
+                        .column( 10 )
+                        .data()
+                        .reduce( function (a, b) {
+                            let contentA = jQuery(a).val();
+                            let contentB = jQuery(b).val();
+                            return intVal(contentA) + intVal(contentB);
+                        }, 0 );
+
+                    var hoursLeft = api
+                        .column( 11 )
+                        .data()
+                        .reduce( function (a, b) {
+                            let contentA = jQuery(a).val();
+                            let contentB = jQuery(b).val();
+                            return intVal(contentA) + intVal(contentB);
+                        }, 0 );
+
+                    var loggedHours = api
+                        .column( 12 )
+                        .data()
+                        .reduce( function (a, b) {
+                            return intVal(a) + intVal(b);
+                        }, 0 );
+
+
+                    // Update footer by showing the total with the reference of the column index
+                    jQuery( api.column( 9 ).footer() ).html('Total');
+                    jQuery( api.column( 10 ).footer() ).html(plannedHours);
+                    jQuery( api.column( 11 ).footer() ).html(hoursLeft);
+                    jQuery( api.column( 12 ).footer() ).html(loggedHours);
+
+                },
```

Comment:
Any way we can easily move this to HTMX rather than adding more JavaScript?

Reply:
hm unfortunately I don't think that is possible in this case. This is part of the table initialization. It creates a sum at the bottom of the table for datatables. With column visibility being a frontend only picker it would probably break. I can check when I create a table component in the future